internals/daemon/api_checks_test.go

```diff
@@ -221,3 +221,61 @@ func (s *apiSuite) postChecks(c *C, body string) *resp {
 	c.Check(rec.Code, Equals, rsp.Status)
 	return rsp
 }
+
+func (s *apiSuite) TestPostChecksRefresh(c *C) {
+	writeTestLayer(s.pebbleDir, `
+checks:
+    chk1:
+        override: replace
+        level: ready
+        exec:
+            command: echo "hello"
```

Comment:
How do we know the check was actually run/refreshed? Should we have it create a file in a temp dir that we assert exists, like we do in some other tests?